features/step_definitions/features_steps.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+Given('a feature {string} of {provider}') do |feature_name, provider|
+  @service_plan = provider.service_plans.where(name: "Basic").last
+  feature = @service_plan.service.features.find_or_create_by(name: feature_name, scope: "ServicePlan")
+end
+
+Given('an enabled feature {string} of {provider}') do |feature_name, provider|
+  @service_plan = provider.service_plans.where(name: "Basic").last
+  feature = @service_plan.service.features.find_or_create_by(name: feature_name, scope: "ServicePlan")
+  @service_plan.features << feature
+  @service_plan.save!
+end
+
+Then('there {is} no feature named {string}') do |is, string|  
+  feature_name = @service_plan.service.features.last.name
+  assert_not_equal feature_name, string
+end
+
+Then('there {is} feature named {string}') do |is, string|
+  feature_name = @service_plan.service.features.last.name
+  assert_equal feature_name, string
+end
+
+And('I disable feature {string}') do |string|
+  element = find('.fa-check-circle')
+  element.click
+end
+
+And('I enable feature {string}') do |string|
+  element = find('.fa-times-circle')
+  element.click
+end
+
+Then('feature {string} should be enabled') do |string|
+  feature = @service_plan.service.features.find_by(name: string)
+  assert_equal 1, feature.features_plans.count
+end
+
+Then('feature {string} should be disabled') do |string|
+  using_wait_time(10) do
```

Comment:
So, I checked some other features with a similar behavior (AJAX calls that result in updating the UI), this is one example: https://github.com/3scale/porta/blob/master/features/old/api/metrics/visibility.feature#L72-L80

You see, its steps use `wait_for_requests` helper method https://github.com/3scale/porta/blob/master/features/step_definitions/metric_steps.rb#L144-L147

Also, you can use this feature's (metrics enable/disable) steps as an example to improve this step - rather than checking the models directly we need to check how enabling/disabling the metric gets reflected in the UI (by changing the icon).

Also, I found some interesting information here: https://github.com/teamcapybara/capybara#asynchronous-javascript-ajax-and-friends 
So, when we are querying with Capybara we also need to take care of the expressions we use to make sure the retry logic is triggered.

And finally, Rails 6 upgrade PR changes the Capybara's default wait time from 2 to 10, so maybe it will also help going forward: https://github.com/3scale/porta/blob/THREESCALE-7405-rails-6/features/support/capybara.rb#L31